website/frontend/views/api_test.py

```diff
@@ -99,3 +99,35 @@ def test_api_v1_download_with_id(self):
         response = self.client.get("/api/v1/download/?id=addrelease")
         self.assert200(response)
         self.assertEquals(response.content_type, 'application/zip')
+
+    # /v1/releases
+
+    def test_api_v1_releases(self):
+        "Test /api/vi/releases"
+        response = self.client.get("/api/v1/releases")
+        self.assert200(response)
+        self.assertEquals(type(response), dict)
```

Comment:
Wouldn't this be response.json?

Reply:
Of course.  I knew I was missing something.